index.html

```diff
@@ -220,7 +218,71 @@ <h2>About me</h2>
         <li>API Design</li>
         <li>Web Development</li>
       </ul>
-      
     </section>
+    <section>
+      <h3 id="form-heading">
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </h3>
+      <form action="https://formspree.io/f/myyqezwd" method="POST">
+        <ul class="form-list">
+          <li class="input-border">
+            <input
+              type="text"
+              name="name"
+              id="name"
+              maxlength="30"
+              placeholder="Full name"
+            />
+          </li>
+          <li class="input-border">
+            <input
+              type="email"
+              name="email"
+              placeholder="Email address"
+              id="email"
+            />
+          </li>
+          <li>
+            <textarea
+              name=""
+              id=""
+              cols="30"
+              rows="5"
+              placeholder="Write me something..."
+              maxlength="500"
+            ></textarea>
```

Comment:
- [ ] Good job adding all inputs, however, you are still missing the part of the requirements that states `A user cannot submit the form if one or more of the fields is empty.` This means that you have to make all input fields `required`